website/layouts/_default/baseof.html

```diff
@@ -5,46 +5,62 @@
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
-        crossorigin="anonymous">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
-        crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
+        integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
     <link rel="stylesheet" href="{{ "/main.css" | relURL }}" />
     <link rel="stylesheet" href="{{ "/syntax.css" | relURL }}" />
     <link rel="icon" href="{{ "/icon-light.png" | relURL }}" />
     <title>{{ block "title" . }}{{ .Site.Title }}{{ end }}</title>
-    <meta name="description" content="{{ .Site.Params.Description }}"></meta>
+    <meta name="description" content="{{ .Site.Params.Description }}">
+    </meta>
     <meta property="og:image" content="{{ "/icon-light.png" | absURL }}" />
     <meta property="og:type" content="object" />
     <meta property="og:title" content="Thanos" />
     <meta property="og:description" content="{{ .Site.Title }}" />
-    <meta name="twitter:card" content="summary_large_image"></meta>
-    <meta name="twitter:site" content="@{{ .Site.Params.TwitterHandle }}"></meta>
-    <meta name="twitter:title" content="Thanos Metrics"></meta>
-    <meta name="twitter:description" content="{{ .Site.Params.Description }}"></meta>
-    <meta name="twitter:image" content="{{ "/thanos-twitter.png" | absURL }}"></meta>
+    <meta name="twitter:card" content="summary_large_image">
+    </meta>
+    <meta name="twitter:site" content="@{{ .Site.Params.TwitterHandle }}">
+    </meta>
+    <meta name="twitter:title" content="Thanos Metrics">
+    </meta>
+    <meta name="twitter:description" content="{{ .Site.Params.Description }}">
+    </meta>
+    <meta name="twitter:image" content="{{ "/thanos-twitter.png" | absURL }}">
+    </meta>
 </head>
 
 <body>
     <nav class="navbar navbar-expand-lg navbar-dark bg-purple">
-        <a href="{{ "/" | relURL }}" class="navbar-brand"><img src="{{ "/icon-dark.png" | relURL }}" alt="Thanos logo" /> Thanos</a>
+        <a href="{{ "/" | relURL }}" class="navbar-brand"><img src="{{ "/icon-dark.png" | relURL }}"
+                alt="Thanos logo" /> Thanos</a>
```

Comment:
Splitting up HTML tags over multiple line makes it a bit harder to read IMO, especially when it's not every single attribute on a newline. Can we tidy this up?

Reply:
Alright.
I'm on it 💯